src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef

```diff
@@ -55,6 +55,37 @@
 ; WSL Port supplier
 "2"="{267B1341-AC92-44DC-94DF-2EE4205DD17E}"
 
+; Registration to use lldb with the port suppliers
+[$RootKey$\AD7Metrics\Engine\{5D630903-189D-4837-9785-699B05BEC2A9}]
+@="MI Debug Engine - lldb"
+"Attach"=dword:00000001
+; "AddressBP"=dword:00000001
+"AlwaysLoadLocal"=dword:00000001
+; "CallStackBP"=dword:00000001
+; "ConditionalBP"=dword:00000001
+"DataBP"=dword:00000001
```

Comment:
```suggestion
"DataBP"=dword:00000000
```
`BreakWatch` is not implemented for `lldb`.

https://github.com/microsoft/MIEngine/blob/69512399f0278d37d96a2b91d4818cf1c2ecd549/src/MICore/CommandFactories/MICommandFactory.cs#L501